ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -550,7 +1236,8 @@ objects:
 
         # begin bootstrapping
         if [[ "${CLUSTER_TYPE}" == "aws" ]]; then
-          RHCOS_AMI=ami-0df3f99538fbef10f  # FIXME: assumes AWS_REGION is us-east-1
+          # RHCOS_AMI=ami-0df3f99538fbef10f  # 4.1 ami FIXME: assumes AWS_REGION is us-east-1
+          RHCOS_AMI=ami-0ae2df22579e00be5  # 4.2 ami FIXME: assumes AWS_REGION is us-east-1
```

Comment:
Is it normal for AWS UPI jobs to require this?

Reply:
Yes, I'm not sure if this value could be provided via a parameter but that would be more ideal.
It was required to bump it from the 4.1 AMI because it didn't have the logic to update the machine's trusted cert.

Reply:
> Yes, I'm not sure if this value could be provided via a parameter but that would be more ideal.

Other platforms use `rhcos.json` (e.g. [here][1]).  But I dislike using an installer-internal file for our CI, and would rather stay closer to [our official docs][2] to ensure we bump AMIs in the docs when we make some change that breaks vs. the currently-documented AMIs.  But this is complicated by us documenting different AMIs for different 4.y, and I don't have an easy way around that yet.  I'd really like openshift/installer#2092 or one of those family to give users the same access to the RHCOS JSON that we enjoy in CI.

[1]: https://github.com/openshift/release/blob/94c18abe89089d386644056095f30a0b61d69ca5/ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml#L520
[2]: https://docs.openshift.com/container-platform/4.2/installing/installing_aws_user_infra/installing-aws-user-infra.html#installation-aws-user-infra-rhcos-ami_installing-aws-user-infra

Reply:
>  I'm not sure if this value could be provided via a parameter but that would be more ideal.

I guess providing via a job-specific parameter would be the way to get per-4.y AMIs in in the absence of something like openshift/installer#2092.